src/screens/Airdrop/RequirementsScreen.tsx

```diff
@@ -0,0 +1,160 @@
+import { StackNavigationProp } from '@react-navigation/stack';
+import React, { Component } from 'react';
+import { Image, Text, StyleSheet, View, TouchableOpacity } from 'react-native';
+import { connect } from 'react-redux';
+
+import { icons } from 'app/assets';
+import { ScreenTemplate, Button, Header } from 'app/components';
+import { Route, MainCardStackNavigatorParams } from 'app/consts';
+import { CONST } from 'app/consts/models';
+import { ApplicationState } from 'app/state';
+import { actions } from 'app/state/airdrop';
+import { ThankYouFlowCompleted } from 'app/state/airdrop/actions';
+import { typography, palette } from 'app/styles';
+
+const i18n = require('../../../loc');
+
+interface ActionProps {
+  completeThankYouFlow: () => ThankYouFlowCompleted;
+}
+
+type Props = {
+  navigation: StackNavigationProp<MainCardStackNavigatorParams, Route.AirdropRequirements>;
+} & ActionProps;
+
+class AirdropRequirementsScreen extends Component<Props> {
+  onSoundsGreatPress = () => {
+    this.props.completeThankYouFlow();
+    // TODO: this.props.navigation.navigate(Route.Dashboard);
+  };
+
+  onTermsConditionsPress = () => {
+    // TODO: this.props.navigation.navigate(Route.AirdropTermsAndConditions);
+  };
+
+  getRequirementsListItems = () =>
+    Object.values(i18n.airdrop.requirements.points).map((requirement, i) => (
+      <View key={i} style={styles.listItem}>
+        <Image source={icons.roundTick} style={styles.image} />
+        <Text style={styles.listText}>{requirement}</Text>
+      </View>
+    ));
+
+  render() {
+    return (
+      <ScreenTemplate
+        header={<Header isBackArrow title={i18n.airdrop.title} />}
+        footer={
+          <>
+            <Button
+              testID="sounds-great"
+              onPress={this.onSoundsGreatPress}
+              style={styles.soundsGreatButton}
+              title={i18n.airdrop.requirements.soundsGreat}
+            />
+            <View style={styles.termsAndConditions}>
+              <Text style={styles.description}>{i18n.airdrop.requirements.termsAndConditions.read}</Text>
+              <TouchableOpacity onPress={this.onTermsConditionsPress}>
+                <Text style={styles.termsText}>{i18n.airdrop.requirements.termsAndConditions.termsAndConditions}</Text>
+              </TouchableOpacity>
+            </View>
+          </>
+        }
+      >
+        <Text style={styles.subtitle}>{i18n.airdrop.requirements.subtitle}</Text>
+        <Text style={styles.description}>
+          {i18n.formatString(i18n.airdrop.requirements.description, CONST.airdropTotalDollarsToShare)}
```

Comment:
`{i18n.formatString(i18n.airdrop.requirements.description, { airdropTotalDollarsToShare: CONST.airdropTotalDollarsToShare)}}` would be nice to add meaningful name here so the translators would know what they are transleting. Later in the translations:
`description:
This conversation was marked as resolved by jellyfish-tom
        'We are glad that you are a part of it! We want to share ${airdropTotalDollarsToShare} in BTCV as a reward for being a supportive community.',`

Reply:
done